process-controller/src/main/java/org/jboss/as/process/logging/ProcessLogger.java

```diff
@@ -289,15 +289,15 @@
      *
      * @return the message.
      */
-    @Message(id = Message.NONE, value = "Keep a copy of the persistent domain configuration even if this host is not the Domain Controller")
+    @Message(id = Message.NONE, value = "Keep a complete copy of the persistent domain configuration even if this host is not the Domain Controller. This parameter will override any ignore-unused-configuration setting in host.xml and attempt to store the complete domain configuration.")
```

Comment:
Is this the semantic? I think --backup controls the default behavior when ignore-unused-configuration is not set, but if it is set, the setting applies.

This allows a slave to keep a copy of the relevant parts of the domain wide config locally for its own use when the DC is down. It doesn't need that copy to have parts that are irrelevant to its servers.